Dockerfile.dapper

```diff
@@ -18,11 +19,13 @@ RUN apt-get -q update && \
     curl -Lo /usr/bin/kubectl https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/linux/${ARCH}/kubectl && \
     chmod a+x /usr/bin/kubectl && \
     curl -sfL https://install.goreleaser.com/github.com/golangci/golangci-lint.sh | sh -s -- -b $(go env GOPATH)/bin ${LINT_VERSION} && \
-    curl https://storage.googleapis.com/kubernetes-helm/helm-v2.14.0-linux-amd64.tar.gz | tar -xzf - && \
-    cp linux-${ARCH}/helm /usr/bin/ && \
-    curl -Lo /usr/bin/kind https://github.com/kubernetes-sigs/kind/releases/download/v0.3.0/kind-linux-${ARCH} && chmod a+x /usr/bin/kind && \
+    curl "https://storage.googleapis.com/kubernetes-helm/helm-${HELM_VERSION}-linux-${ARCH}.tar.gz" | tar -xzf - && \
+    cp linux-${ARCH}/helm /usr/bin/ && chmod a+x /usr/bin/helm && \
+    curl -LO "https://github.com/kubernetes-sigs/kubefed/releases/download/v${KUBEFED_VERSION}/kubefedctl-${KUBEFED_VERSION}-linux-${ARCH}.tgz" && \
+    tar -xzf kubefedctl-${KUBEFED_VERSION}-linux-${ARCH}.tgz && cp kubefedctl /usr/bin/ && chmod a+x /usr/bin/kubefedctl && \
+    curl -Lo /usr/bin/kind "https://github.com/kubernetes-sigs/kind/releases/download/${KIND_VERSION}/kind-linux-${ARCH}" && chmod a+x /usr/bin/kind && \
     go get -v github.com/onsi/ginkgo/ginkgo && go get -v github.com/onsi/gomega/... && \
-    go get golang.org/x/tools/cmd/goimports
+    go get -v golang.org/x/tools/cmd/goimports
```

Comment:
Just a question out of curiosity (not related to this patch, and no action required):

is there any big advantage in doing it all in one big run with && .. && .. && 
beyond avoiding several layers? Is the layer overhead too big?

It's a little bit annoying to wait for the whole container being rebuilt for a simple change :) :-)

Reply:
We can create more layers with separate RUN, but each later is 1.4 GB. We wanted to minimize the number of layers. The alternative is bunch of 1.4 GB layers laying around :).

Reply:
IN any case we should host the build image on some public docker registry and pull it only once, this will improve the times considerably without the need to wait for dependencies on every change.

Reply:
1.4GB! :-o why that much? :D

Reply:
apt-get update

Reply:
@mangelajo After some testing you are right, better solution would be separate each install step in to its own RUN command. Storage is less expensive then time :).

Reply:
Thanks!! :)